config/config.exs

```diff
@@ -33,8 +33,8 @@ config :sentry,
   }
 
 config :statix,
-  host: {:system, "STATSD_HOST", "localhost"},
-  port: {:system, "STATSD_PORT", 8125, {String, :to_integer}}
+  host: {:system, "DOGSTATSD_HOST", "localhost"},
```

Comment:
what if we kept the datadog standard? `DD_HOSTNAME`  https://docs.datadoghq.com/agent/docker/?tab=standard#global-options